src/server/app.js

```diff
@@ -0,0 +1,42 @@
+const config = require('./config');
+const express = require('express');
+const Server = require('./my-server');
+const createRoutes = require('./routes');
+const morgan = require('morgan');
+
+class App {
+    constructor ({ port = config.PORT, host = config.HOST } = {}) {
+        this.port = port;
+        this.host = host;
+    }
+
+    async _configExpress (e) {
+        const routes = await createRoutes(config.ROUTES_DIR);
+
+        e.use(routes);
+
+        if (config.NODE_ENV === 'development')
+            e.use(morgan('dev'));
```

Comment:
I thought about this for a long time, but I would like to see all logging through the `debug` module. However it is questionable, we can discuss it in private.